packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -20,12 +25,46 @@ const WG_DATA = {
   name: 'membership',
 }
 
+const WG_OPENING_DATA = {
+  title: 'Membership worker role',
+  shortDescription: 'Lorem Ipsum...',
+  description: 'Bigger Lorem ipsum...',
+  applicationDetails: 'Application process default',
+  applicationFormQuestions: [
+    { question: '🐁?', type: OpeningMetadata.ApplicationFormQuestion.InputType.TEXT },
+    { question: '🐘?', type: OpeningMetadata.ApplicationFormQuestion.InputType.TEXTAREA },
+  ],
+  hiringLimit: 1,
```

Comment:
2. Change the `hiringLimit` here instead:
```suggestion
  hiringLimit: 5,
```
(so the actual input field should be tested)